bpf/bpf_lxc.c

```diff
@@ -50,6 +50,20 @@ static __always_inline bool redirect_to_proxy(int verdict, __u8 dir)
 }
 #endif
 
+#ifdef ENABLE_CUSTOM_CALLS
+static __always_inline void
+encode_custom_prog_meta(struct __ctx_buff *ctx, __u32 ret, int direction,
+			int identity)
+{
+	__u32 custom_meta = 0;
+
+	custom_meta += (identity & 0xffffff);
+	custom_meta += (ret & 0xff) << 24;
+	custom_meta += (direction & 0x1) << 31;
```

Comment:
Nit: I find the use of `+=` a bit misleading here, maybe `|=`?

Reply:
Also, `0x80000000` is a little ambiguous as it can be set by either the `ret` or the `direction`. Although in practice I think only low positive integer values are likely to be passed in `ret` for the forseable future, it probably makes sense to have the `ret` use a mask of `0xEF`?

Reply:
Agreed for the `|=`, and the `0xff` instead of `0xef` is totally a mistake on my side. I'll fix it, thanks.